x/stakeibc/keeper/unbonding_records.go

```diff
@@ -74,6 +74,14 @@ func (k Keeper) CreateEpochUnbondingRecord(ctx sdk.Context, epochNumber uint64)
 	return true
 }
 
+// Helper function to evaluate if a host zone unbonding record still needs to be initiated
+func (k Keeper) ShouldInitiateHostZoneUnbondingRecord(ctx sdk.Context, hostZoneRecord *recordstypes.HostZoneUnbonding) bool {
```

Comment:
thoughts on renaming to `ShouldHostZoneRecordUnbond`?

Reply:
Imo the proposed change is more clear.